atom/app/atom_main_delegate_mac.mm

```diff
@@ -61,6 +61,11 @@
   base::mac::SetBaseBundleID(base_bundle_id.c_str());
 }
 
+std::string AtomMainDelegate::GetSystemPreferredLanguage() {
```

Comment:
Where does this get used?

Reply:
Ah, I think it got lost somewhere in the rebase. I'll update.